README.md

```diff
@@ -6,106 +8,64 @@
 
 1. What is the first step for installing Dart on a Windows machine?
 
-A) Install Homebrew
-B) Download the Dart SDK
-C) Update your PATH
-D) Run Dart Doctor
-
+'C) Update your PATH '
 
```

Comment:
This is the first step

Reply:
B)

Reply:
B) Download the Dart SDK.
After downloading the SDK, you'll proceed with updating your PATH and running other setup steps.

Reply:
B

Reply:
Download the Dart SDK